exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTx.java

```diff
@@ -60,34 +57,23 @@
 
   /**
    * Creates a create wallet transaction from its message.
```

Comment:
Here and elsewhere: not technically a message, but serialized transaction data, or serialized transaction parameters/arguments _from_ the transaction message.